src/libraries/Microsoft.PowerFx.Core/Functions/Blob/UriBlob.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    public class UriBlob : BlobElementBase
+    {        
+        public Uri Uri { get; }
+
+        public UriBlob(Uri uri)            
+        {            
+            Uri = uri ?? throw new ArgumentNullException(nameof(uri));
+        }
+
+        public override async Task<byte[]> GetAsByteArrayAsync(CancellationToken token)
+        {
+            token.ThrowIfCancellationRequested();
+
+            if (Uri.Scheme == "https")
+            {
+                using HttpClient httpClient = new HttpClient();
```

Comment:
>using HttpClient httpClient = new HttpClient(); [](http://example.com/codeflow?start=16&length=47)

I imagine at some future point we may need a service for this. 
Potentially to auth; but also for lifespan management of HttpClients.

Reply:
I'll remove this class